lib/random.py

```diff
@@ -0,0 +1,28 @@
+"""Generate pseudo random numbers. Should not be used for security purposes"""
```

Comment:
Please add a period at the end of the end:

"""Generate pseudo random numbers. Should not be used for security purposes."""